src/components/TankCard.js

```diff
@@ -0,0 +1,75 @@
+import styled from 'styled-components';
+
+export default function Card({ deleteTank, title, text, fish, img }) {
+  return (
+    <CardContainer>
+      <h1>{title}</h1>
```

Comment:
h1 is not the right heading level I think when there could be more than one tank card rendered on this page